python/lsst/consdb/hinfo.py

```diff
@@ -161,10 +162,10 @@ def process_column(column_def: str | Sequence, info: dict) -> Any:
         The value to use for the column.
         None if any input value is missing.
     """
-    if type(column_def) is str:
+    if isinstance(column_def, str):
```

Comment:
Should check is not None ?

Reply:
This can't be `None` if it passes mypy.  When we move the configuration outside to a YAML file, Pydantic will validate it.